src/apps/dashboard/routes/scheduledtasks/index.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect } from 'react';
+import Page from 'components/Page';
+import globalize from 'lib/globalize';
+import Box from '@mui/material/Box';
+import Stack from '@mui/material/Stack';
+import { QUERY_KEY, useTasks } from '../../features/scheduledtasks/api/useTasks';
+import { getCategories, getTasksByCategory } from '../../features/scheduledtasks/utils/tasks';
+import Loading from 'components/loading/LoadingComponent';
+import Tasks from '../../features/scheduledtasks/components/Tasks';
+import type { TaskInfo } from '@jellyfin/sdk/lib/generated-client/models/task-info';
+import serverNotifications from 'scripts/serverNotifications';
+import Events, { Event } from 'utils/events';
+import { ApiClient } from 'jellyfin-apiclient';
+import { useApi } from 'hooks/useApi';
+import { queryClient } from '../../../../utils/query/queryClient';
+
+const ScheduledTasks = () => {
+    const { __legacyApiClient__ } = useApi();
+    const { data: tasks, isLoading } = useTasks({ isHidden: false });
+
+    // TODO: Replace usage of the legacy apiclient when websocket support is added to the TS SDK.
+    useEffect(() => {
+        const onScheduledTasksUpdate = (_e: Event, _apiClient: ApiClient, info: TaskInfo[]) => {
+            queryClient.setQueryData([QUERY_KEY], info);
+        };
+
+        const fallbackInterval = setInterval(() => {
+            if (!__legacyApiClient__?.isMessageChannelOpen()) {
+                void queryClient.invalidateQueries({
+                    queryKey: [QUERY_KEY]
+                });
+            }
+        }, 1e4);
+
+        __legacyApiClient__?.sendMessage('ScheduledTasksInfoStart', '1000,1000');
```

Comment:
We do have constants for the message types in the SDK: https://typescript-sdk.jellyfin.org/enums/generated-client.SessionMessageType.html#scheduledtasksinfo